DESCRIPTION

```diff
@@ -18,4 +18,5 @@ LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.0
 Suggests: 
-    testthat
```

Comment:
Just curious as to why you are not specifying a version constraint for `data.table` or `zoo`?

Reply:
ah! because I forgot 😬 thanks for pointing that out.